src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -60,9 +60,8 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 	                    uint8_t occupancyMask[][kDisplayWidth + kSideBarWidth], bool drawUndefinedArea = true);
 	bool renderSidebar(uint32_t whichRows, uint8_t image[][kDisplayWidth + kSideBarWidth][3],
 	                   uint8_t occupancyMask[][kDisplayWidth + kSideBarWidth]);
-	void displayAutomation();
-	void displayParameterName(int32_t paramID);
-	void setDisplayParameterNameTimer();
```

Comment:
new functions to handle the rendering of the display and update of automation values and led indicators in automation view